src/api/app/controllers/webui/comments_controller.rb

```diff
@@ -1,8 +1,13 @@
 class Webui::CommentsController < Webui::WebuiController
   before_action :require_login
-  before_action :find_commentable, only: :create
+  before_action :set_commented, only: :create
 
   def create
+    if @commented.nil?
+      flash.now[:error] = "Failed to create comment: This #{@commentable_type.name.downcase} does not exist anymore."
+      return render partial: 'layouts/webui/flash'
```

Comment:
I prefer it this way:
```suggestion
      render partial: 'layouts/webui/flash' and return
```
This solution is described in the Rails guides, we don't need the method `create` to return what `render` returns, and I don't see the benefit to shorten the code for readibility. See also [this explanation](https://codereview.stackexchange.com/questions/996/return-render-in-ruby-on-rails/997#997). What do you think?

Reply:
Oh sorry, I completely overlooked the link. I'll change it now

Reply:
You are completely right, I didn't like it very much. Indeed it seemed to return something when in fact it didn't